R/pkg/R/group.R

```diff
@@ -229,6 +229,20 @@ gapplyInternal <- function(x, func, schema) {
   if (is.character(schema)) {
     schema <- structType(schema)
   }
+  arrowEnabled <- sparkR.conf("spark.sql.execution.arrow.enabled")[[1]] == "true"
```

Comment:
use `sparkR.conf(key, defaultValue)` instead? otherwise it will call `stop()` if the config is not set?

Reply:
Actually, for this specifically, unlike other configurations, `sparkR.conf` is guaranteed to return a default value for the configuration that exists in `SQLConf` if I didn't misread. It should be okay to leave the default value in `SQLConf`.